### PR TITLE
DP-64: Add meta descriptions to each page in the operation guide.

### PR DESCRIPTION
**Pull request by Bonrob2**
Added description tags for device configuration backup and performance management sections of the operation>deep-dive guide as part of SEO improvment.

### All Contributors

* [ ] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [ ] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### Contribution Checklist

* Please [make an issue in the OpenNMS issue tracker](https://opennms.atlassian.net/) if there isn't one already.<br />Once there is an issue, please:
  1. update the title of this PR to be in the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the Jira link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
  4. once you've created this PR, please link to it in a comment in the Jira issue
  Don't worry if this sounds like a lot, we can help you get things set up properly.
* **If this code is likely to affect the UI, did you name your branch with `-smoke` in it to trigger smoke tests?**
* If this is a new or updated feature, is there documentation for the new behavior?
* If this is new code, are there unit and/or integration tests?
* If this PR targets a `foundation-*` branch, does it try to avoid changing files in `$OPENNMS_HOME/etc/`?

### What's Next?

A PR should be assigned at least 2 reviewers.  If you know that someone would be a good person to review your code, feel free to add them.

If you need help making additions or changes to the documentation related to your changes, please let us know.

In any case, if anything is unclear or you want help getting your PR ready for merge, please don't hesitate to say something in the comments here,
or in [the #opennms-development chat channel](https://chat.opennms.com/opennms/channels/opennms-development).

Once reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the OGP) you are welcome to merge the PR when you're ready.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* Jira (Issue Tracker): https://opennms.atlassian.net/browse/DP-64

### CODE REVIEW

File: docs/modules/operation/pages/deep-dive/device-config-backup/configuration.adoc [hunk 1]
```diff
@@ -1,6 +1,7 @@
 
 [[poller-config]]
 = Poller Configuration
+:description: Poller configuration in OpenNMS Horizon/Meridian for device configuration backup: TFTP port and IP address, events, parameters, thread pool adjustments.
```
Comment (mmahacek) [inline docs/modules/operation/pages/deep-dive/device-config-backup/configuration.adoc:4]:
```suggestion
:description: Device Configuration Backup feature poller configuration in OpenNMS Horizon/Meridian: TFTP port and IP address, events, parameters, thread pool adjustments.
```

File: docs/modules/operation/pages/deep-dive/device-config-backup/introduction.adoc [hunk 2]
```diff
@@ -1,5 +1,6 @@
 
 = Device Configuration Backup
+:description: Overview of how to back up configuration for devices like routers, firewalls, and switches with OpenNMS Horizon/Meridian, for better disaster recovery.
```
Comment (mmahacek) [inline docs/modules/operation/pages/deep-dive/device-config-backup/introduction.adoc:3]: Can also be utilized for "change management", if we're looking to enhance page metadata

File: docs/modules/operation/pages/deep-dive/performance-data-collection/data-types.adoc [hunk 3]
```diff
@@ -1,6 +1,7 @@
 
 [[data-types]]
 = Data Types
+:description: Over of data types collected in OpenNMS Horizon/Meridian including counter, gauge, integer, IpAddress, OctetString, Opaque, String, and TimeTicks.
```
Comment (mmahacek) [inline docs/modules/operation/pages/deep-dive/performance-data-collection/data-types.adoc:4]:
```suggestion
:description: Overview of data types collected in OpenNMS Horizon/Meridian including counter, gauge, integer, IpAddress, OctetString, Opaque, String, and TimeTicks.
```

File: docs/modules/operation/pages/deep-dive/performance-data-collection/shell/adhoc-collection.adoc [hunk 4]
```diff
@@ -1,6 +1,7 @@
 
 [[ga-performance-mgmt-adhoc-collection]]
 = Ad hoc collection
+:description: opennms:collect Karaf Shell command in OpenNMS Horizon/Meridian to trigger and perform a collection on any of the available collectors.
```
Comment (mmahacek) [inline docs/modules/operation/pages/deep-dive/performance-data-collection/shell/adhoc-collection.adoc:4]:
```suggestion
:description: Ad hoc collection commands in Horizon/Meridian via Karaf shell to perform a collection on any of the available collectors for troubleshooting and testing.
```

File: docs/modules/operation/pages/deep-dive/performance-data-collection/shell/collectd-schedule.adoc [hunk 5]
```diff
@@ -2,6 +2,7 @@
 [[ga-performance-mgmt-schedule-shell]]
 
 = Collectd Schedule
+:description: opennms:pollerd-schedule Karaf Shell command to output details of the Collectd scheduler.
```
Comment (mmahacek) [inline docs/modules/operation/pages/deep-dive/performance-data-collection/shell/collectd-schedule.adoc:5]:
```suggestion
:description: Overview of Karaf shell commands in OpenNMS Horizon/Meridian to help administer and diagnose issues related to performance data collection scheduling.
```

File: docs/modules/operation/pages/deep-dive/device-config-backup/introduction.adoc (hunk 2 above)
Comment (Bonrob2) [inline docs/modules/operation/pages/deep-dive/device-config-backup/introduction.adoc:3]:
```suggestion
:description: Back up configuration for devices like routers, firewalls, switches with OpenNMS Horizon/Meridian, for better disaster recovery and change management.
```

**Review by mmahacek — APPROVED** [review]

**Pull request merged**